doc/modules/feature_extraction.rst

```diff
@@ -318,7 +318,7 @@ corpus of text documents::
 
   >>> corpus = [
   ...     'This is the first document.',
-  ...     'This is the second second document.',
+  ...     'This is the second document.',
```

Comment:
I think we need keep this here for now. The [CI test](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=37991&view=logs&j=dde5042c-7464-5d47-9507-31bdd2ee0a3a&t=cbc19d79-e4aa-510d-a75f-976447ff9ab7) are failing because the docs depend the second "second".